Sources/AppcuesKit/Presentation/Extensions/UIBezierPath+Tooltip.swift

```diff
@@ -15,84 +15,118 @@ internal struct Pointer {
 
     let edge: Edge
     let size: CGSize
+    let cornerRadius: CGFloat
     let offset: CGFloat
 }
 
-extension UIBezierPath {
-    convenience init(tooltipAround mainRect: CGRect, cornerRadius: CGFloat, pointer: Pointer) {
-        self.init()
+extension CGPath {
+    static func tooltip(around mainRect: CGRect, cornerRadius: CGFloat, pointer: Pointer) -> CGPath {
+        let path = CGMutablePath()
 
-        let triangle = Triangle(pointer: pointer, mainRect: mainRect, cornerRadius: cornerRadius)
+        let triangle = CGMutablePath.Triangle(pointer: pointer, mainRect: mainRect, cornerRadius: cornerRadius)
 
         // Draw the path clockwise from top left
```

Comment:
I noticed all the `clockwise` params switched from true to false, is this still an accurate comment?

Reply:
We're still drawing clockwise in terms of the order of sides and corners. It seems like CGPath draws differently from UIBezierPath, and so just changing this value made it work, but I'm not sure I can explain why. I was going to revisit these angles but forgot. Will take a closer look.

Reply:
Yeah, UIKit and Quartz have different coordinate systems. UIKit left, Quartz right:
![image](https://user-images.githubusercontent.com/845681/226356984-d0b471d0-3135-4548-90ae-109e4b6e37dc.jpeg)

https://www.informit.com/articles/article.aspx?p=2149190&seqNum=11

I also noticed `addRelativeArc` is an option that's simpler to use, so switching to that since it doesn't have clockwise options at all. Still not sure why `clockwise: false` was correct.

Reply:
ah ok, didn't know that. thanks for the info.